pages/linux/apache2ctl.md

```diff
@@ -0,0 +1,24 @@
+# apache2ctl
+
+> The CLI tool to administrate HTTP web server Apache.
+> This commmand comes with Debian based OS, for RHEL based OS see httpd.
+
+- Start the Apache daemon.  Give an error if it is already running:
+
+`apache2ctl start`
+
+- Stop the Apache daemon:
+
+`apache2ctl stop`
+
+- Restart the Apache daemon:
+
+`apache2ctl restart`
+
+- Check configuration syntax file:
```

Comment:
"Test syntax of the configuration file" - to match with the flag `-t`. We try to include a word which starts with the flag.